### GOAL DEFINITION
uint8[] target
uint8 ACTION_CALIBRATE=1
uint8 ACTION_MOVE_NAME=2
uint8 ACTION_MOVE_CELL=3
uint8 POSENAME_STOW=1
uint8 POSENAME_HOME=2
uint8 POSENAME_HOVER=3
---
### RESULT DEFINITION
string data
---
### FEEDBACK DEFINITION
string data
